packages/wallet/backend/src/paymentPointer/service.ts

```diff
@@ -413,6 +426,136 @@ export class PaymentPointerService implements IPaymentPointerService {
       throw new NotFound()
     }
 
+    if (paymentPointer.isWM) {
+      await this.deps.cache.set(paymentPointer.id, paymentPointer, {
+        expiry: 60
+      })
+    }
+
     return paymentPointer
   }
+
+  async processWMPaymentPointers(): Promise<void> {
```

Comment:
This is the only method that needs to be reviewed. The other changes are from Dragos' PR.